src/core/math/vec3/index.js

```diff
@@ -19,10 +19,10 @@ module.exports = {
   min: require('./min'),
   multiply: require('./multiply'),
   negate: require('./negate'),
-  normal: require('./normal'),
+//  normal: require('./normal'),
   normalize: require('./normalize'),
-  randomNonParallel: require('./randomNonParallel'),
-  rotate: require('./rotate'),
+  random: require('./random'),
```

Comment:
looking at the code for 'randomNonParallel' it seems not random at all , we need a better name

Reply:
I’ll review the old code. I believe this is used by plane, and needs to be random, i.e. making a new vector randomly.